ui/page/privacy/manual_mixer_setup_page.go

```diff
@@ -105,78 +104,130 @@ func NewManualMixerSetupPage(l *load.Load, dcrWallet *dcr.Asset) *ManualMixerSet
+									return label.Layout(gtx)
 								}),
 								layout.Rigid(func(gtx C) D {
-									return layout.Inset{Top: values.MarginPaddingMinus15}.Layout(gtx, func(gtx C) D {
-										return pg.mixerAccountSections(gtx, values.String(values.StrUnmixedAccount), func(gtx layout.Context) layout.Dimensions {
-											return pg.unmixedAccountSelector.Layout(pg.ParentWindow(), gtx)
-										})
-									})
-								}),
-								layout.Rigid(func(gtx C) D {
-									return layout.Inset{Top: values.MarginPadding10, Left: values.MarginPadding16, Right: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
-										return layout.Flex{
-											Axis: layout.Horizontal,
-										}.Layout(gtx,
-											layout.Rigid(func(gtx C) D {
-												return pg.Theme.Icons.ActionInfo.Layout(gtx, pg.Theme.Color.Gray1)
-											}),
-											layout.Rigid(func(gtx C) D {
-												txt := `<span style="text-color: grayText2">
-											<b>Make sure to select the same accounts from the previous privacy setup. </b><br>Failing to do so could compromise wallet privacy.<br> You may not select the same account for mixed and unmixed.
-										</span>`
-												return layout.Inset{
-													Left: values.MarginPadding8,
-												}.Layout(gtx, renderers.RenderHTML(txt, pg.Theme).Layout)
-											}),
-										)
-									})
+									label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleWarningDesc))
+									return label.Layout(gtx)
 								}),
 							)
-						}),
+						})
+					}),
+				)
+			})
+		})
+	})
+}
+
+func (pg *ManualMixerSetupPage) backLayout(gtx C) D {
+	return layout.Inset{Right: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
+		// Setting a minimum Y larger than the label allows it to be centered.
+		gtx.Constraints.Min.Y = int(values.MarginPadding40)
+		return layout.Flex{Alignment: layout.Start}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Inset{
+					Left:  values.MarginPadding5,
+					Right: values.MarginPadding10,
+				}.Layout(gtx, func(gtx C) D {
+					return layout.Flex{
+						Axis:      layout.Vertical,
+						Alignment: layout.Middle,
+						Spacing:   layout.SpaceAround,
+					}.Layout(gtx,
 						layout.Rigid(func(gtx C) D {
-							gtx.Constraints.Min.X = gtx.Constraints.Max.X
-							return layout.UniformInset(values.MarginPadding15).Layout(gtx, pg.toPrivacySetup.Layout)
+							return pg.backIcon.Layout(gtx, values.MarginPadding30)
 						}),
 					)
 				})
-			},
-		}
-		return page.Layout(pg.ParentWindow(), gtx)
-	}
-
-	return cryptomaterial.UniformPadding(gtx, body)
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle, Spacing: layout.SpaceAround}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						label := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleManualTitle))
+						return layout.E.Layout(gtx, label.Layout)
+					}),
+				)
+			}),
+		)
+	})
 }
```

Comment:
too many unnecessary Layouts, making the code more complicated
You can try the code below

```
func (pg *ManualMixerSetupPage) backLayout(gtx C) D {
	gtx.Constraints.Min.X = gtx.Constraints.Max.X
	// Setting a minimum Y larger than the label allows it to be centered.
	gtx.Constraints.Min.Y = int(values.MarginPadding50)
	return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
		layout.Rigid(func(gtx C) D {
			return layout.Inset{
				Left:  values.MarginPadding15,
				Right: values.MarginPadding15,
			}.Layout(gtx, func(gtx C) D {
				return pg.backIcon.Layout(gtx, values.MarginPadding30)
			})
		}),
		layout.Rigid(func(gtx C) D {
			return layout.Center.Layout(gtx, func(gtx C) D {
				return pg.Theme.H6(values.String(values.StrSetUpStakeShuffleManualTitle)).Layout(gtx)
			})
		}),
	)
}
```

Reply:
Ok.